grt/main.go

```diff
@@ -240,36 +254,39 @@ nextIssue:
 		printIssues(w, other, markdownLinks)
 		fmt.Fprintf(w, "\n")
 	}
+	return nil
 }
 
-func createRelease(ctx context.Context, client *github.Client, owner, repo, milestone, name string, close, pre bool, changelog string) {
+func createRelease(ctx context.Context, client *github.Client, owner, repo, release string, changelog string) error {
+	splits := strings.SplitN(release, "-", 2)
+	milestone := splits[0]
+	pre := len(splits) == 2
```

Comment:
```suggestion
	pre := release != milestone
```

(feels more straightforward and possibly future-safe though the result should be identical?)